evm_loader/program/src/entrypoint.rs

```diff
@@ -556,6 +560,33 @@ fn process_instruction<'a>(
                 _ => return Err!(ProgramError::InvalidAccountData),
             };
 
+            let backend = SolanaBackend::new(&account_storage, Some(trx_accounts));
+            debug_print!("  backend initialized");
+
+            let executor = Machine::restore(&storage, backend);
+            debug_print!("Executor restored");
+
+            let executor_state = executor.into_state();
+            let used_gas = executor_state.substate().metadata().gasometer().used_gas();
+
+            let (gas_limit, gas_price) = storage.get_gas_params()?;
+            if used_gas > gas_limit {
+                return Err!(ProgramError::InvalidArgument);
+            }
+            let gas_price_wei = U256::from(gas_price);
+            let fee = U256::from(used_gas)
+                .checked_mul(gas_price_wei).ok_or_else(||E!(ProgramError::InvalidArgument))?;
+
+            let (caller_info, caller_token_info) = account_storage.get_caller_account_info().ok_or_else(||E!(ProgramError::InvalidArgument))?;
```

Comment:
Why user token is being getting here, but not mentioned in accounts like in `Continue` or `CallFromRawEthereumTX`

Reply:
fixed